import-automation/executor/app/executor/import_executor.py

```diff
@@ -569,12 +608,17 @@ def _upload_import_inputs(
                 dest=dest,
             )
 
-        self.uploader.upload_string(
-            version,
-            os.path.join(output_dir, self.config.storage_version_filename))
-        self.uploader.upload_string(
-            self._import_metadata_mcf_helper(import_spec),
-            os.path.join(output_dir, self.config.import_metadata_mcf_filename))
+        if self.config.ignore_validation_status or validation_status:
```

Comment:
Wouldn't this check skip upload if validation fails and ignore_validation_status is False?
I assume ignore_validation_status is set when  latest_version.txt is to be updated without validations.